### PR TITLE
portuguese (pt-br) language translation revision

### PR DESCRIPTION
**Pull request by dprado75**
Main changes:
- fixed mistype in locale_config.js file
- "cenário" renamed to "cena" to better support its original meaning.
- "entrar" renamed to "fazer login" as this is the usual expression used in a log in form ("Entrar" was kept as the title).
- a whole text block was missing translation and it is now completed. 
- minor fixes and spell checking.

### CODE REVIEW

**Comment by dprado75** [comment]
Hi @keianhzo Thanks for reviewing the files. You can reply this or tag me if you need any further information on this pull request. The locale_config.js file fixes a mistype ("Português" is the correct spelling for native speakers) and ads "pt-pt" to address Portuguese from Portugal users.
The pt-bt.json file its already commented on the original request.

File: src/assets/locales/locale_config.js [hunk 1]
```diff
@@ -15,5 +15,6 @@ export const FALLBACK_LOCALES = {
   "zh-hans": "zh",
   "zh-hans-cn": "zh",
   pt: "pt-br",
+  "pt-pt": "pt-br",
```
Comment (keianhzo) [inline src/assets/locales/locale_config.js:18]: He have updated the code to handle the sub-language tags so we don't need this anymore. Rebase from master and keep the current master code when rebasing. Let me know if you have issues with that and I can rebase and create a new PR for this.

**Review by keianhzo — CHANGES_REQUESTED** [review]
Remember to also do a `node sync_locales.js`from `hubs/src/assets/locales` before pushing.

**Comment by keianhzo** [comment]
@dprado75 I've moved this PR to https://github.com/mozilla/hubs/pull/3526 to save you the time of rebasing and updating this branch. Thanks for the contribution!

**Pull request closed without merging**